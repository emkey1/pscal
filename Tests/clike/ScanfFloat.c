int main() {
    float f;
    scanf(f);
    printf("read: ");
    printf(f);
    printf("\n");
    return 0;
}
