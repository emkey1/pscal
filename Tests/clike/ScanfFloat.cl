int main() {
    float f;
    scanf(f);
    printf("read: %f\n", f);
    return 0;
}
