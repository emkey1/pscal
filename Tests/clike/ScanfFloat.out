read: 3.140000
